app/controllers/public/speaker/view.js

```diff
@@ -0,0 +1,9 @@
+import Controller from '@ember/controller';
+import { action, computed } from '@ember/object';
```

Comment:
Codacy found an issue: ['action' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403732)